### PR TITLE
fix(theme): fix theme package lookup

### PR DESCRIPTION
**Pull request by rbardini**
Normalize theme package name before passing to program actions, and delegate module resolution to Node.

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -8,6 +8,10 @@ var colors = require('colors');
 var chalk = require('chalk');
 var path = require('path');
 
+function getThemeName(theme) {
+  return theme.match('jsonresume-theme-.*') ? theme : `jsonresume-theme-${theme}`;
+}
```
Comment (antialias) [inline index.js:13]: If `getThemeName` could be factored into a [custom options processor](https://www.npmjs.com/package/commander#custom-option-processing), the value would be computed once and stored on `program`. This way, `getThemeName` would not need to be invoked on lines 48 and 57 below.

Comment (rbardini) [reply index.js:13]: Good idea, will give it a try 👍

Comment (rbardini) [reply index.js:14]: Done.

**Comment by hkdobrev** [comment]
I believe this PR broke a documented behaviour of the `resume serve` command.

When developing a theme one should be available to run `resume serve` inside the theme folder and that should load `index.js` from the current folder. Currently, it loads the default theme.

**Pull request merged**